mods/cnc/languages/lua/en.ftl

```diff
@@ -0,0 +1,147 @@
+new-secondary-objective = New secondary objective
+
+# cnc64gdi01
+destroy-obelisk-sams = Destroy the SAM sites protecting the Obelisk.
+destroy-obelisk = Destroy the Obelisk.
+destroy-biotech = Destroy the biotech facility.
+
+## cnc64gdi01, gdi03, gdi05abc, gdi07 gdi08ab, gdi09
+destroy-sams = Destroy the SAM sites to receive air support.
+
+## funpark01
+investigate-village = Investigate the nearby village for reports of
+    strange activity.
+reach-village = Reach the village.
+kill-creatures = Kill all creatures in the area.
+
+## gdi01
+establish-beachhead = Establish a beachhead
+
+## gdi01, gdi02, gdi03, gdi04c, gdi05abc, nod07a, nod08ab, nod09, nod10b
+eliminate-nod = Eliminate all Nod forces in the area.
+
+## gdi02
+capture-refinery = Capture the Tiberium refinery.
+
+## gdi02, gdi04c, gdi05abc, nod01
+destroy-gdi = Destroy all GDI troops.
+
+## gdi04ab
+battlefield-control = Battlefield Control
+apcs-left = APC squads in reserve: { $apcs }
+retrieve-rods = Retrieve the crate with the stolen rods.
+eliminate-reinforcements = Eliminate { $kills } Nod units for reinforcements.
+
+## gdi04c
+defend-bialystok = Defend the town of Białystok.
+
+## gdi05abc
+find-gdi-base = Find the GDI base.
+
+## gdi06
+destroy-nod-building = Destroy the Nod ********.
+
+## gdi07
+destroy-nod = Destroy remaining Nod structures and units.
+construct-base = Construct all available buildings.
+
+## gdi08a
+repair-base-vehicles = Repair GDI base and vehicles.
+
+## gdi08a, gdi09
+destroy-nod-force = Destroy the Nod strike force.
+
+## gdi08b
+protect-mobius = Protect Dr. Mobius.
+protect-hospital = Protect the Hospital.
+keep-civilians-alive = Keep at least { $civilians } out of 14 Civilians alive.
+destroy-nod-bases = Destroy the Nod bases.
+
+## gdi08ab, gdi09, nod02ab, nod05, nod07c
+kill-all-enemies = Kill all enemies!
+
+## gdi09
+destroy-nod-bunkers = Destroy the Nod bunkers to allow Carter's
+    convoy to pass through safely.
+convoy-passed-partly = Part of Carter's convoy passed through!
+
+## nod01
+kill-nikoomba = Kill Nikoomba.
+destroy-village = Destroy the village.
+eliminate-nod-forces = Eliminate all Nod forces.
+
+## nod02a
+destroy-gdi-units = Destroy all GDI units.
+
+## nod02ab
+build-base = Build a base.
+
+## nod02b, nod05
+destroy-gdi-base = Destroy the GDI base.
+
+## nod03ab
+capture-prison = Capture the prison.
+destroy-gdi-forces = Destroy all GDI forces.
+
+## nod04a
+destroy-village-kill-civilians = Destroy the village and kill all civilians.
+kill-gdi-units = Kill all GDI units in the area.
+
+## nod04b
+protect-nod-supporters = Protect the civilians that support Nod.
+kill-gdi-supporters = Kill all civilian GDI supporters.
+
+## nod05, nod07ab, nod09
+build-sams = Build { $sams } SAMs to fend off the GDI bombers.
+
+## nod06abc
+steal-nuclear-detonator = Steal the GDI nuclear detonator.
+move-to-evacuation-point = Move to the evacuation point.
+
+## nod06c
+destroy-gdi-supporter-houses = Destroy the houses of the GDI supporters
```

Comment:
this is not what I had in mind. `destroy-gdi-supporter-houses` is used by `nod06a` and `nod06b`, but not `nod06c`